install/kubernetes/quick-install.yaml

```diff
@@ -619,6 +620,7 @@ spec:
           {}
       hostNetwork: true
       restartPolicy: Always
+      priorityClassName: system-cluster-critical
```

Comment:
@aanm what's the story with this setting again? I thought we've been trying to avoid setting it on the quick-install for some reason.

Reply:
@joestringer this will be fixed before 1.8 release. I've created this issue https://github.com/cilium/cilium/issues/10504